src/components.js

```diff
@@ -204,85 +206,44 @@ export default {
             ],
         },
     },
-    Icon: {
```

Comment:
Please make sure there are no changes to existing components. This branch should only add 2 new components to this file.

Reply:
Sorry about that, looks like some merging issues.

Although I see that on `default-list` branch some components are listed two times: `Icon`, `DefaultList`, `DefaultListItem`, `ScrollableList`, `Typeahead`, `TypeaheadMultiselect` and `ColumnFilter`. We should probably fix that there.

Reply:
I fixed and merged `default-list` branch, thanks for pointing that out.

Reply:
Now only `PieChart` and `InlineDialog` are duplicated. Sorry for missing these two previously :)

Reply:
I don't see those two being duplicated